front/src/modules/pathfinding/utils.ts

```diff
@@ -176,3 +176,21 @@ export const isStation = (chCode: string): boolean =>
   chCode === 'BV' || chCode === '00' || chCode === '';
 
 export const isPathStepInvalid = (step: PathStep | null): boolean => step?.isInvalid || false;
+
+export function mergeDuplicatedOperationalPoints<T>(
```

Comment:
This function could deserve some documentation. Here is a suggestion. Does that work for you @Akctarus @clarani ?
```suggestion
/**
 * Ensure that the array doesn't have any duplicated operational points by merging the duplicates.
 * They usually can be differentiated by the track name.
 * @param operationalPoints the list of operational points
 * @param getDuplicatedOP a function that returns a key base on some op properties
 * @param mergeOP a function checking that previous and next op differ by one property and merge this property into the previous op
 * @returns the updated list with unique operational points
 */
export function mergeDuplicatedOperationalPoints<T>(
```

Reply:
done